scenes/GeigerCounter.tscn

```diff
@@ -0,0 +1,88 @@
+[gd_scene load_steps=14 format=2]
+
+[ext_resource path="res://scripts/GeigerViewport.gd" type="Script" id=1]
+[ext_resource path="res://fonts/Montserrat-Medium.ttf" type="DynamicFontData" id=2]
+[ext_resource path="res://fonts/Cutive_Mono/CutiveMono-Regular.ttf" type="DynamicFontData" id=3]
+[ext_resource path="res://geigercounter.ogg" type="AudioStream" id=4]
```

Comment:
do we need this file added to the repo?

Reply:
Yes, it's a scene file that specifies the Geiger Counter object and all its children nodes. There are a lot of `.tscn` files already in the repo, so this isn't out of the ordinary.

Reply:
I actually was referring to the OGG sound file.  I think it's missing

Reply:
I didn't add the `.ogg` sound file because I didn't want to include large asset files (such as `.csv`, `.obj`) in the repo. Nevertheless, `geigercounter.ogg` is only 6kB so I guess I can add it if it's better that way.